pkg/ebpf/c/common/buffer.h

```diff
@@ -98,6 +104,45 @@ statfunc int save_bytes_to_buf(args_buffer_t *buf, void *ptr, u32 size, u8 index
     return 0;
 }
 
+statfunc int load_str_from_buf(args_buffer_t *buf, char *str, u8 index)
+{
+    u16 offset;
+    u32 size;
+
+    // skip if index is not in buffer
+    if (index > buf->argnum)
+        return 0;
+
+    offset = buf->args_offset[index];
+
+    if (offset == INVALID_ARG_OFFSET)
+        return 0;
+
+    // Ensure there is enough space for read index (u8)
+    if (offset + sizeof(u8) > ARGS_BUF_SIZE)
```

Comment:
Even though we're acquainted to operator precedence, I would enclose it to always communicate what we want explicitly. But I don't know if the current linter settings agree with me. Would you mind checking it?

```suggestion
    if ((offset + sizeof(u8)) > ARGS_BUF_SIZE)
```

Reply:
valid point! checked and the current linter settings accepted both. tks